app/app/static/custom/ourstyle.css

```diff
@@ -0,0 +1,132 @@
+/* Chrome, Safari, Edge, Opera */
```

Comment:
Add a comment around at the top of this file to let everyone know that custom CSS should be added here